### ISSUE TITLE
No changelog

### ISSUE THREAD

**Issue by adamchainz** [keleshev/schema#104]
When upgrading from 0.4.0 to 0.5.0, I couldn't find a changelog, so I had to kind of guess what the user-affecting changes were from the commit log on Github. Would be nice to have a changelog included in the readme and on PyPI.

**Comment by skorokithakis** [comment]
Good point, I use python-semantic-release that generates very nice-looking changelogs, but they require prepending commits with "feat/fix/docs/etc". I'll try to do that from now on, good point.

**Comment by ankostis** [comment]
(great project)
Adding a  *manual* changelog is a good idea because you may explicitly list breaking changes and known limitations, and convey precise messages that are relevant to the end-users; commit massages are dev-oriented.
